ESSArch_Core/frontend/lang/en/general.js

```diff
@@ -188,5 +188,6 @@ angular.module('essarch.language').config(function($translateProvider) {
     YOUHAVELOGGEDOUT: 'You have logged out.',
     en: 'English',
     sv: 'Svenska',
+    'This field is required': 'This field is required',
```

Comment:
This could be moved to a new file, `forms.js`, which includes all general translations for forms